packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -48,14 +55,47 @@ class MiniCart extends Component {
 
     get productList() {
         const { cartId, cartCurrencyCode, cart } = this.props;
-        return cartId ? (
+        return cartId && !this.state.editPanelOpen ? (
             <ProductList
                 currencyCode={cartCurrencyCode}
                 items={cart.details.items}
             />
         ) : null;
     }
 
+    get editPanel() {
```

Comment:
If possible, this should use the existing `drawer` abstraction. If that's unsuitable, let's talk about why, so we don't have lots of duplication.

Reply:
The ability to edit items in the cart was left out since this pull was becoming pretty large. The `editPanel` here isn't even used. I'll just remove it for now.